src/main/java/Duke.java

```diff
@@ -1,72 +1,162 @@
+                userCommand = userInput.nextLine();
+                continue;
+            } else if (userCommand.contains("event")) {
+                inputCount = printEvent(arrayInput, userCommand, inputCount);
+                taskType[inputCount - 1] = "E";
                 userCommand = userInput.nextLine();
                 continue;
             }
-            arrayInput[inputCount++] = userCommand;
-            System.out.println(line);
-            System.out.println("added: " + userCommand);
-            System.out.println(line);
+            inputCount = printUpdate(arrayInput, userCommand, inputCount);
             userCommand = userInput.nextLine();
         }
 
         if (userCommand.equals("bye")) {
-            System.out.println(line);
-            System.out.println("Bye. Hope to see you again soon!");
-            System.out.println(line);
+            printBye();
         }
+    }
 
+    private static int printEvent(String[] arrayInput, String userCommand, int inputCount) {
+        String at;
+        int index = userCommand.indexOf("-");
+        at = userCommand.substring(index - 5);
+        Event event = new Event(userCommand, at, index);
+        inputCount++;
+        System.out.println(line);
+        System.out.println("Got it. I've added this task:");
+        System.out.println(event);
+        System.out.println("Now you have " + inputCount + " tasks in the list.");
+        System.out.println(line);
+        String taskName = event.description.substring(6, index - 10);
+        String eventName = taskName + " (at: " + at + ")";
+        arrayInput[inputCount - 1] = eventName;
+        return inputCount;
+    }
+
+    private static int printDeadline(String[] arrayInput, String userCommand, int inputCount) {
+        String by;
+        int index = userCommand.indexOf("by");
+        by = userCommand.substring(index + 3);
+        Deadline deadline = new Deadline(userCommand, by, index);
+        inputCount++;
+        System.out.println(line);
+        System.out.println("Got it. I've added this task:");
+        System.out.println(deadline);
+        System.out.println("Now you have " + inputCount + " tasks in the list.");
+        System.out.println(line);
+        String taskName = deadline.description.substring(9, index - 2);
+        String deadlineName = taskName + " (by: " + by + ")";
+        arrayInput[inputCount - 1] = deadlineName;
+        return inputCount;
+    }
+
+    private static int printTodo(String[] arrayInput, String userCommand, int inputCount) {
+        Todo todo = new Todo(userCommand);
+        inputCount++;
+        System.out.println(line);
+        System.out.println("Got it. I've added this task:");
+        System.out.println(todo);
+        System.out.println("Now you have " + inputCount + " tasks in the list.");
+        System.out.println(line);
+        arrayInput[inputCount - 1] = todo.description.substring(5);
+        return inputCount;
+    }
```

Comment:
printDeadline(), printEvent() and printTodo() can possibly be merged into one function.